netcore/Makefile.am

```diff
@@ -3,7 +3,7 @@
 #
 
 # Extracted MicrosoftPrivateCoreFxNETCoreAppVersion from https://github.com/dotnet/coreclr/blob/master/eng/Versions.props#L11
-NETCORETESTS_VERSION := 4.6.0-preview5.19209.12
+NETCORETESTS_VERSION := 4.6.0-preview5.19215.16
```

Comment:
Could you please fix both `Versions.props#L11` to be L12 and L14? 🙂 in the comments

Reply:
Yes